### ISSUE TITLE
function get_loss_source_joint may be error

### ISSUE THREAD

**Issue by jingzhengli** [yaoyueduzhen/DADA#3]
Thanks for opening the source code~ I have some issues in a function as follows:
the dimension of 'outputs_classifier_s1, outputs_classifier_s2' is 2×C, however the 'labels_source' is C dimension. Thus how to computer class_criterion(outputs_classifier_s1, labels_source)? It will occur an error

   **def get_loss_source_joint(self, inputs_source, labels_source):
        class_criterion = nn.CrossEntropyLoss().cuda()
        _, _, outputs_classifier_s1, outputs_classifier_s2 = self.c_net(inputs_source)
        classifier1_loss_s = class_criterion(outputs_classifier_s1, labels_source)
        classifier2_loss_s = class_criterion(outputs_classifier_s2, labels_source)
        return classifier1_loss_s, classifier2_loss_s**

**Comment by jingzhengli** [comment]
> Thanks for opening the source code~ I have some issues in a function as follows:
> the dimension of 'outputs_classifier_s1, outputs_classifier_s2' is 2×C, however the 'labels_source' is C dimension. Thus how to computer class_criterion(outputs_classifier_s1, labels_source)? It will occur an error
> 
> **def get_loss_source_joint(self, inputs_source, labels_source): class_criterion = nn.CrossEntropyLoss().cuda() _, _, outputs_classifier_s1, outputs_classifier_s2 = self.c_net(inputs_source) classifier1_loss_s = class_criterion(outputs_classifier_s1, labels_source) classifier2_loss_s = class_criterion(outputs_classifier_s2, labels_source) return classifier1_loss_s, classifier2_loss_s**
Sorry, It is my fault. In function class_criterion, the label is not ONE-HOT.
